langchain/chains/qa_with_sources/__init__.py

```diff
@@ -28,7 +28,7 @@ def _load_stuff_chain(
     document_variable_name: str = "summaries",
     **kwargs: Any,
 ) -> StuffDocumentsChain:
-    llm_chain = LLMChain(llm=llm, prompt=prompt)
+    llm_chain = LLMChain(llm=llm, prompt=prompt, verbose=kwargs.get("verbose", False))
```

Comment:
given the amount of times we do `kwargs.get()`, lets maybe just make it an explicit kwarg in thsi function, eg `verbose=False` on line 29

Reply:
Thanks for the suggestion, made verbose an explicit kwarg.